unilog.go

```diff
@@ -260,6 +261,13 @@ func (u *Unilog) handleError(action string, e error) {
 		return
 	}
 
+	if Stats != nil {
+		name := fmt.Sprintf("unilog.errors.%s", action)
+		name = strings.Replace(name, " ", "_", -1)
+		emsg := strings.Replace(e.Error(), " ", "_", -1)
```

Comment:
This might be bad for metric cardinality - we should check what errors we can get here (Also, ought to replace everything that's not alphanumeric; tags are pretty sensitive to stuff)